AppiumLibrary/keywords/_element.py

```diff
@@ -94,6 +94,13 @@ def hide_keyboard(self, key_name=None):
         driver = self._current_application()
         driver.hide_keyboard(key_name)
 
+    def is_keyboard_shown(self):
+        """Return true if Android keyboard is displayed or False if not displayed
+        No parameters are used.
+        """
+        driver = self._current_application()
+        return driver.is_keyboard_shown()

```

Comment:
is this not working on ios? Have you tested, cuase if `hide_keyword` works this should work right?

Reply:
This keyword is only Android not for iOS